src/api/app/models/package.rb

```diff
@@ -1320,7 +1320,10 @@ def abort_build(params)
 
   def backend_build_command(command, params)
     begin
-      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{params.to_query}", '')
+      # Note: This list needs to keep in sync with the backend code
+      permitted_params = params.permit(:repository, :arch, :package, :code, :wipe)
```

Comment:
Would prefer to do this in controller layer?
However, not sure from where and how often we call backend_build_command ...